base/src/com/google/idea/blaze/base/project/BlazeProjectOpenProcessor.java

```diff
@@ -73,7 +73,7 @@ public boolean canOpenProject(VirtualFile file) {
 
   @Override
   public boolean isStrongProjectInfoHolder() {
-    return Registry.is("bazel.project.auto.open.if.present", false);
+    return Registry.is("bazel.project.auto.open");
```

Comment:
I'm unsure if the change here from true to false is expected

Reply:
Are you suggesting to keep the extra registry key for this ProjectOpenProcessor?

Reply:
yes I believe we should keep the original key here as is, but just drop 2nd arg